.github/workflows/docker-push-to-registries.yml

```
@@ -215,23 +265,25 @@ jobs:
 
   trivy-scan-image:
     needs: build
-    uses: telicent-oss/shared-workflows/.github/workflows/trivy-image-scan.yml@main
+    uses: telicent-oss/shared-workflows/.github/workflows/trivy-image-scan.yml@update-workflow-for-docker-with-build-artifact
```

```suggestion
    uses: telicent-oss/shared-workflows/.github/workflows/trivy-image-scan.yml@main
```
Pin back to `main` prior to merging